pkg/cri/server/sandbox_run.go

```diff
@@ -350,23 +351,36 @@ func (c *criService) RunPodSandbox(ctx context.Context, r *runtime.RunPodSandbox
 }
 
 // setupPodNetwork setups up the network for a pod
-func (c *criService) setupPodNetwork(ctx context.Context, sandbox *sandboxstore.Sandbox) error {
+func (c *criService) setupPodNetwork(ctx context.Context, sandbox *sandboxstore.Sandbox) (retErr error) {
```

Comment:
There are some other reasons the pod network init might error out and some additional latency on the setup see https://github.com/containerd/containerd/pull/6046/files#diff-ed79a44f7781f44137f81c80f65985ecb11ddd0a68b854e33aa9ebb47897acd7R115-R167  latency mainly on the newNS for the network ns for this pod.

Reply:
Added as a separate metric since sandbox creation is different from sandbox configuration.